objectherkenning_openbare_ruimte/on_edge/detection_pipeline/components/data_detection.py

```diff
@@ -0,0 +1,345 @@
+import csv
+import logging
+import os
+import pathlib
+import shutil
+import time
+from typing import List
+
+import numpy as np
+import torch
+from cvtoolkit.helpers.file_helpers import delete_file
```

Comment:
If we copy this single function to `on_edge/utils.py` we won't have the dependency on cvtoolkit for the code on the device, so we also don't have to "hack" it into the docker file.

Reply:
After splitting the repo it will make sense, now there is some cvt_toolkit stuff used in the not on edge code.